code/modules/power/lighting/light.dm

```diff
@@ -88,6 +88,17 @@
 /obj/machinery/light/Initialize(mapload)
 	. = ..()
 
+	// Detect and scream about double stacked lights
+	if(PERFORM_ALL_TESTS(focus_only/stacked_lights))
+		var/turf/our_location = get_turf(src)
+		for(var/obj/machinery/light/on_turf in our_location)
+			if(on_turf == src)
+				continue
+			if(on_turf.dir != dir)
+				continue
+			stack_trace("Conflicting double stacked light [on_turf.type] found at ([our_location.x],[our_location.y],[our_location.z])")
+			qdel(on_turf)
```

Comment:
Why're we deleting the turf?

Reply:
![image](https://user-images.githubusercontent.com/34697715/221438144-5ec0fa2f-5563-48e1-92b1-7d10867c74af.png)

whar

Reply:
oh lol i misread

Reply:
means it might be a bad name